plugin/trino-hive/src/main/java/io/trino/plugin/hive/aws/athena/PartitionProjectionProperties.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive.aws.athena;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+
+import static io.trino.plugin.hive.aws.athena.projection.Projection.invalidProjectionException;
+import static java.lang.String.format;
+
+public final class PartitionProjectionProperties
+{
+    /**
+     * General properties suffixes
+     */
+    static final String PROPERTY_SUFFIX_TABLE_PROJECTION_ENABLED = "enabled";
```

Comment:
nit: COLUMN_PROJECTION_TYPE_PROPERTY_SUFFIX or just COLUMN_PROJECTION_TYPE_SUFFIX reads better

Reply:
Applied to all consts of this kid: `COLUMN_PROJECTION_TYPE_SUFFIX `